Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10-alpine
+
+WORKDIR /app
+
```

Comment:
Add: _**COPY package*.json ./**_
Best practice because then it will create a layer of node modules and we'll avoid re-building node-modules every change of the source code